misk-aws/src/main/kotlin/misk/jobqueue/sqs/ResolvedQueue.kt

```diff
@@ -13,7 +13,8 @@ internal class ResolvedQueue(
   val url: String,
   val region: AwsRegion,
   val accountId: AwsAccountId,
-  val client: AmazonSQS
+  val client: AmazonSQS,
+  val maxRetries: Int,
```

Comment:
Could a default value be set here? I anticipate this will cause build failures when it rolls out due to Misk version mismatch with downstream callers constructing this object.